src/main/msp/msp_protocol.h

```diff
@@ -62,7 +62,7 @@
 #define MSP_PROTOCOL_VERSION                0
 
 #define API_VERSION_MAJOR                   1  // increment when major changes are made
-#define API_VERSION_MINOR                   46 // increment after a release, to set the version for all changes to go into the following release (if no changes to MSP are made between the releases, this can be reverted before the release)
+#define API_VERSION_MINOR                   47 // increment after a release, to set the version for all changes to go into the following release (if no changes to MSP are made between the releases, this can be reverted before the release)
```

Comment:
Good catch. We forgot to bump on release. Not sure why version is not in line with 4.6 as long I'm here.

Reply:
The versions are independent. It's simply causality that both are similar.

Reply:
Perhaps we can correct this when we finally bump MAJOR version

Reply:
@haslinghuis the theory is that we must increase the MAJOR version for each release with changes, and the MINOR if there is some bug in the release that modifies some of the MSP elements.

But for error, we are increasing the MINOR with each release, and the MAJOR when the MSP protocol changed from V1 to V2 (we have the MSP_PROTOCOL_VERSION for this).

This was discovered too late, and for this reason was not changed in the past. But now we are ignoring a little the backwards compatibility in the protocol, so maybe we can fix it in a future.